src/main/flight/failsafe.c

```diff
@@ -65,11 +65,11 @@ PG_REGISTER_WITH_RESET_TEMPLATE(failsafeConfig_t, failsafeConfig, PG_FAILSAFE_CO
 PG_RESET_TEMPLATE(failsafeConfig_t, failsafeConfig,
     .failsafe_throttle = 1000,                       // default throttle off.
     .failsafe_throttle_low_delay = 100,              // default throttle low delay for "just disarm" on failsafe condition
-    .failsafe_delay = 4,                             // 0,4sec
-    .failsafe_off_delay = 10,                        // 1sec
+    .failsafe_delay = 10,                                // 1 sec, can regain control on signal recovery, at idle in drop mode
```

Comment:
Rationale for the increase in stage 1 time to 1s, not 0.4s:
With Steve's fixes, we go to stage1 reliably at 300ms, and, by default this means we get idle throttle, and start falling, at 300ms.  If we regain Rx within the 1s stage1 time, we could immediately regain control, and perhaps fly home.
This seems preferable to the old situation, where an Rx loss for more than 0.4s would enter a hard disarm, inevitably causing a crash. 
Pilots flying expensive gear may elect to set a longer stage1 time, eg 2s, in the hope that they will recover signal and avoid a crash.  They may elect use an intermediate stage1 throttle value to delay the rate of fall.

Reply:
I don't like the 1 sec default here. This could serious cause harm on drop mode leaving the drone 1 sec in the air. Otherwise failsafe is working on Crossfire as tested previous.

Reply:
I should point out that the previous 400ms stage 1 period started at the end of the 300ms hold period.  There was a bug in which throttle was being 'held' for the whole of this time.  

In "Drop" mode, with earlier 4.3 versions, the motors typically would stay locked for 700ms after Rx signal loss.

Now, after exactly 300ms, motor drive falls to idle, and the quad will start to fall from that point.  In "Drop" mode, the motors will stop 700ms later.  The time to total motor stop is only 300ms longer than before, and this improves our chances of recovery.  Motors will be at idle for all except the first 300ms of that time.  I think that is a better safety outcome than before.

If we wanted it to be exactly the same as before, we would set the failsafe period too 0.7s.